frontend/src/wrapper/TopNavigation.tsx

```diff
@@ -111,81 +112,115 @@ export default function TopNavigation({
           pr: '24px', // keep right padding when drawer closed
         }}
       >
-        <IconButton
-          edge='start'
-          color='inherit'
-          aria-label='open drawer'
-          onClick={toggleDrawer}
-          sx={{
-            marginRight: 2,
-            ...(drawerOpen && { display: 'none' }),
-          }}
-        >
-          <MenuIcon />
-        </IconButton>
-        <Box sx={{ flexGrow: 1, ml: 2 }}>
-          <Box sx={{ width: 'max-content', cursor: 'pointer', px: 1 }}>
-            <Link href='/beta' color='inherit' underline='none' style={{ color: 'inherit', textDecoration: 'inherit' }}>
-              <Typography variant='h5' component='div'>
-                <span style={{ fontFamily: 'Pacifico' }}>Bailo</span>
-                {betaAdornment}
-              </Typography>
+        <Box sx={{ display: { xs: 'flex', sm: 'none' } }}>
+          <IconButton onClick={handleNavMenuClicked}>
+            <MenuIcon sx={{ color: 'white' }} />
```

Comment:
Should be using theme colours

Reply:
Do you mean setting up a custom theme colour with the value of white?